### DISCUSSION TITLE
Bug in the UI Search Box

### DISCUSSION THREAD

**Discussion by osseonews** [oramasearch/orama#831] [Q&A]
You have a bug in your UI Search Box. When you configure the path for the link, it works in the Get a Summary Panel (AI Chat), but not in the Search Results (non-AI keyword search). What happens is that you get an undefined + the link in the Search Results (undefined is appended to the URL). But, in the AI search the link is correct.  Please see attached screen shots

SEARCH SCREEN (see the undefined appended - this is for every link - link doesnt work)
------------------------------------------------------------------
![search_screen_undefined](https://github.com/user-attachments/assets/dd47b203-651e-4f0f-8ea9-9b403f768ecc)

AI RESULTS (link is correct and works)
------------------------------------------------
![ai-results](https://github.com/user-attachments/assets/42d75c0b-8f1b-4441-86a2-f30c81608597)

**Comment by osseonews** [comment]
I tried to ask this in the forums, but get no response. For some reason, on all our indexes we get this "undefined" error. No matter what we do. We have deleted indexes, recreated them. Followed all your instructions. On search we get an undefined, always. We don't see that in your sample on your website, but this happens to us with every index. This is also causing the UI Components to fail in React. There is some bug somewhere. Maybe it has to do with our schema, but we have also just copied your instructions exactly and when we load a schema, we get this undefined everywhere.

**Comment by micheleriva** [comment]
Hi @osseonews, thank you for opening this. We're working on it and will keep you posted 🙏

**Comment by rjborba** [comment]
@osseonews Hi! Thanks for spotting it! It should be fixed today/tomorrow! I'll let you know!
